avssync.go

```diff
@@ -20,7 +20,7 @@ type AvsSync struct {
 	syncInterval                 time.Duration
 	operators                    []common.Address // empty means we update all operators
 	quorums                      []byte
-	fetchQuorumsDynamically      bool
+	noFetchQuorumsDynamically    bool
```

Comment:
why negate this one? 
For binaries that are already running with this flag set to some value, it will have the opposite effect even if the configs are kept the same

Reply:
ehh good catch! I meant to also update the flag and env name of course!
I realized that binary flags' default is false and can't be set to true (other than dynamically in code, which I find ugly and bug prone), so I opted to do it this way because my assumption is that most everyone would want to update all of their quorums automatically without needing to restart avs-sync after adding a new quorum. So 2 questions for you:
- is that a valid assumption?
- if so, are you fine with this change? or do you have another suggestion for how to do this?

Reply:
> my assumption is that most everyone would want to update all of their quorums automatically

I think this assumption is fair. Would it be easier to keep the name the same but change the default to true by making it a `cli.BoolTFlag`?

Reply:
complete brains, I didn't even know `BoolTFlag` was an option! Reverted to this, thanks :) [b5f5b41](https://github.com/Layr-Labs/avs-sync/pull/6/commits/b5f5b41a3a279a4099e9b3c65cefa2610978b1b2)